src/intervals/conversion_promotion.jl

```diff
@@ -2,14 +2,12 @@
 
 ## Conversion and promotion
 
-## Default conversion is to Interval{Float64}
+## Default conversion to Interval, corresponds to Interval{Float64}
 convert{T<:Real}(::Type{Interval}, x::T) = make_interval(Float64, x)
-@compat convert{T<:Union{Float64,BigFloat,Rational}}(::Type{Interval}, x::T) =
-    make_interval(T, x)
 
-## Conversion to intervals (with rounding) from Integer or Irrational
+## Conversion to specific type intervals
 @compat convert{T<:Union{Float64,BigFloat}, S<:Real}(::Type{Interval{T}}, x::S) =
```

Comment:
Despite Lint.jl, there's no need to parametrize `S` explicitly, you can just put `x::Real`, which is clearer in my opinion.